script/os-autoinst-setup-multi-machine

```diff
@@ -17,7 +17,7 @@ ensure_ip_forwarding() {
 
 install_packages() {
     command -v retry > /dev/null || zypper -n in retry
-    retry -- zypper -n in openvswitch os-autoinst-openvswitch firewalld libcap-progs
+    retry -e -s 30 -r 7 -- zypper -n in openvswitch os-autoinst-openvswitch firewalld libcap-progs
```

Comment:
That likely won't prevent the problem. We need to rerun `zypper ref`. We have something similar elsewhere already. I suggest you look up that code and use the same here

Reply:
See https://github.com/os-autoinst/os-autoinst-distri-openQA/pull/170